### PR TITLE
Removed unnecessary code.

### PR DESCRIPTION
**Pull request by TimonPost**
Removed some unnecessary code.

### CODE REVIEW

File: amethyst_network/src/network_socket.rs [hunk 1]
```diff
@@ -204,9 +202,8 @@ where
                             e, raw_event.source
                         ),
                     }
-                }
-                if !matched {
-                    println!("Received packet from unknown source");
+                }else {
```
Comment (azriel91) [inline amethyst_network/src/network_socket.rs]: missing a blank space

Comment (TimonPost) [reply amethyst_network/src/network_socket.rs]: yea I noticed, will be fixing it soon.

File: amethyst_network/src/network_socket.rs [hunk 2]
```diff
@@ -172,27 +172,25 @@ where
...
-                    .expect("Unreachable: Channel will be alive until a stop event is sent");
-            } else if net_connection.state == ConnectionState::Disconnected {
+                {
+                    self.tx
+                        .send(InternalSocketEvent::SendEvents {
+                            target,
+                            events: net_connection.send_buffer_early_read().cloned().collect(),
+                        })
+                        .expect("Unreachable: Channel will be alive until a stop event is sent");
+                } else if net_connection.state == ConnectionState::Disconnected {
```
Comment (udoprog) [inline amethyst_network/src/network_socket.rs]: Formatting mishap?

**Comment by TimonPost** [comment]
Ah that's the reason, strange to have meaningless code and expect others to know the filter is supposed to be there. We should probably move this into an issue.

**Comment by Xaeroxe** [comment]
I disagree regarding the issue.  By the time we implement filtering it might not take this form, and leaving this doesn't really save anyone that much work.

**Comment by TimonPost** [comment]
I would just say merge it and by the time we need to implement filters we add it back. It are just a few lines of code.

**Comment by Xaeroxe** [comment]
@jojolepro requesting your feedback on our feedback :P

**Pull request merged**